README.md

```diff
@@ -46,25 +46,23 @@ import cratedb
 
 ### Connecting to CrateDB
 
-Connect to a CrateDB cluster in the cloud by providing hostname, user name and password:
+Connect to a CrateDB Cloud cluster using SSL, by providing hostname, username, and password:
 
 ```python
 crate = cratedb.CrateDB(
     host="host", 
     user="user", 
-    password="password"
+    password="password",
+    use_ssl=True
```

Comment:
We don't need `use_ssl` by default... by design, as microcontrollers will mostly be connecting to the cloud which requires it so it's on by default to help minimize code size.  Could be a micro-optimization but that was why it's on by default and originally only mentioned in the local network / Docker Crate example.